apps/docs/scripts/pretest.sh

```diff
@@ -1,5 +1,5 @@
 # Check if node is already running at port 4000, if not start it
-# TODO: This is a temporary solution to avoid conflicts with the test node in docs-snippets2
+# TODO: This is a temporary solution to avoid conflicts with the test node in docs-snippets
```

Comment:
This should no longer be necessary, perhaps we can revert this to 8e80ba16a

```bash
# Kill anything running on port 4000
lsof -t -i:4000 | xargs -r kill

# Runs a node at port 4000
pnpm fuels node > /dev/null 2>&1 &

# Builds projects
pnpm fuels build

# Deploys projects (needed for loader bytecode)
pnpm fuels deploy

# Kills the node
lsof -t -i:4000 | xargs -r kill

# Checks for type errors
pnpm tsc --noEmit
```

Although I don't think the type check is necessary anymore post-#3381